Title: 647028 – dev-python/jaraco-logging-1.5 : [TEST] E fixture level_string not found
URL: https://bugs.gentoo.org/show_bug.cgi?id=647028

Go to: 
		Gentoo Home
Documentation
Forums
Lists
Bugs
Planet
Store
Wiki
Get Gentoo!





Gentoo's Bugzilla – Bug 647028
dev-python/jaraco-logging-1.5 : [TEST] E fixture level_string not found
Last modified: 2018-06-30 23:31:26 UTC node [vulture]


Home
| New–[Ex]
| Browse
| Search
| Privacy Policy

| 





[?]
| Reports

| 
Requests

| 
Help


| 
New Account


| 
Log In





[x]



| 
Forgot Password

Login:




[x]













Bug 647028 
      - dev-python/jaraco-logging-1.5 : [TEST] E fixture level_string not found


Summary:
dev-python/jaraco-logging-1.5 : [TEST] E fixture level_string not found
    








Status:
    

RESOLVED
          WORKSFORME
      






Alias:


        None
    





Product:

Gentoo Linux




Classification:

Unclassified




Component:

Current packages

  (show other bugs)



Hardware:

All
        Linux
      







Importance:
      
Normal
       normal
      


Assignee:

Louis Sautier (sbraz)








URL:







Whiteboard:




Keywords:

TESTFAILURE







Depends on:







Blocks:









 





      Reported:
    
2018-02-08 18:19 UTC by Toralf Förster





      Modified:
    
2018-06-30 23:31 UTC
      (History)
    




          CC List:
        

1 
          user
          
            (show)
          



proxy-maint









See Also:






Package list:







Runtime testing required:

---























      Attachments
    




emerge-info.txt

              (emerge-info.txt,16.75 KB,
                text/plain)

            
2018-02-08 18:19 UTC,

            Toralf Förster




Details





dev-python:jaraco-logging-1.5:20180208-180724.log

              (dev-python:jaraco-logging-1.5:20180208-180724.log,8.04 KB,
                text/plain)

            
2018-02-08 18:19 UTC,

            Toralf Förster




Details





emerge-history.txt

              (emerge-history.txt,171.21 KB,
                text/plain)

            
2018-02-08 18:19 UTC,

            Toralf Förster




Details





environment

              (environment,105.05 KB,
                text/plain)

            
2018-02-08 18:19 UTC,

            Toralf Förster




Details





etc.portage.tbz2

              (etc.portage.tbz2,13.05 KB,
                application/x-bzip)

            
2018-02-08 18:19 UTC,

            Toralf Förster




Details





temp.tbz2

              (temp.tbz2,25.48 KB,
                application/x-bzip)

            
2018-02-08 18:19 UTC,

            Toralf Förster




Details





View All

Add an attachment
        (proposed patch, testcase, etc.)
    








Note
              You need to
              log in
              before you can comment on or make changes to this bug.
            

















Description


Toralf Förster






          2018-02-08 18:19:14 UTC
        

file /var/tmp/portage/dev-python/jaraco-logging-1.5/work/jaraco.logging-1.5/jaraco/logging.py, line 14
  def log_level(level_string):
E       fixture 'level_string' not found
>       available fixtures: cache, capfd, caplog, capsys, capturelog, cov, doctest_namespace, environ, metadata, mock, mocker, monkeypatch, pytestconfig, record_xml_property, recwarn, tmpdir, tmpdir_factory, worker_id, workspace
>       use 'pytest --fixtures [testpath]' for help on them.


  -------------------------------------------------------------------

  This is an unstable amd64 chroot image at a tinderbox (==build bot)
  name: 17.0-systemd_abi32+64-test_20180203-182621

  -------------------------------------------------------------------

gcc-config -l:
 [1] x86_64-pc-linux-gnu-7.3.0 *

Available Python interpreters, in order of preference:
  [1]   python3.5
  [2]   python2.7 (fallback)

java-config:
The following VMs are available for generation-2:
*)	IcedTea JDK 3.6.0 [icedtea-bin-8]
Available Java Virtual Machines:
  [1]   icedtea-bin-8  system-vm

emerge -qpv dev-python/jaraco-logging
[ebuild  N    ] dev-python/jaraco-logging-1.5  USE="{-test}" PYTHON_TARGETS="python2_7 python3_5 -python3_4"




Comment 1


Toralf Förster






          2018-02-08 18:19:17 UTC
        

Created attachment 518636 [details]
emerge-info.txt




Comment 2


Toralf Förster






          2018-02-08 18:19:20 UTC
        

Created attachment 518638 [details]
dev-python:jaraco-logging-1.5:20180208-180724.log




Comment 3


Toralf Förster






          2018-02-08 18:19:24 UTC
        

Created attachment 518640 [details]
emerge-history.txt




Comment 4


Toralf Förster






          2018-02-08 18:19:27 UTC
        

Created attachment 518642 [details]
environment




Comment 5


Toralf Förster






          2018-02-08 18:19:30 UTC
        

Created attachment 518644 [details]
etc.portage.tbz2




Comment 6


Toralf Förster






          2018-02-08 18:19:33 UTC
        

Created attachment 518646 [details]
temp.tbz2




Comment 7


Louis Sautier (sbraz)






          2018-06-30 23:31:26 UTC
        

pytest is acting like log_level is a test function which gets passed a fixture called "level_string" (it is not).
It should just be running the doctest. I'll close this because I can't reproduce it even with the same pytest version as you.









Format For Printing
 - XML
 - Clone This Bug
 - Clone In The Same 
                        Product
 - Top of page 







Home
| New–[Ex]
| Browse
| Search
| Privacy Policy

| 





[?]
| Reports

| 
Requests

| 
Help


| 
New Account


| 
Log In





[x]



| 
Forgot Password

Login:




[x]

